app/models/user.rb

```diff
@@ -1,9 +1,14 @@
 class User < ApplicationRecord
+  # has_secure_password
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
:red_circle: As [per this requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md) Your blog-app should implement the `:confirmable` module. Please add and implement this module as it is one of  the project requirements

![requi](https://github.com/maov19/blog-app/assets/26736582/2dd07f53-a5cc-41c5-8ade-972e73063d97)

Reply:
Fixed! Added the confirmable module, as required, both in migration file and models.

Reply:
Hi @maov19,

Great work! :+1:

Please push your changes to GitHub and request for a re-review from your dashboard.

Thanks,